package.json

```diff
@@ -3,30 +3,35 @@
   "version": "0.1.9",
   "description": "Utility functions for working with Quadbins",
   "license": "MIT",
+  "type": "module",
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
   "umd:main": "dist/umd/index.js",
-  "types": "dist/types/index.d.js",
+  "types": "dist/types/index.d.ts",
   "exports": {
     ".": {
+      "types": "./dist/types/index.d.ts",
       "import": "./dist/esm/index.js",
-      "require": "./dist/cjs/index.js",
-      "types": "./dist/types/index.d.ts"
+      "require": "./dist/cjs/index.js"
```

Comment:
The effect of adding `"type": "module"` is that Node.js and various build tools will assume `.js` files are ESM by default, unless otherwise specified by a `.cjs` or `.mjs` extension. Without `"type": "module"`, the assumption for `.js` files is CJS by default.

So, I think we're missing part of the fix here — can we rename the 'require' entrypoint to have a `.cjs` file extension?

Reply:
Example:

https://github.com/donmccurdy/KTX-Parse/blob/1f210187c8ddac75599f31b8a420c69017252a5f/package.json#L5-L15

Reply:
@donmccurdy do you know of a better way to change the file extension (currently using `mv` as I can't figure out how to change behavior of `tsc`)

Reply:
Discussed offline. Blocked in tsc by https://github.com/microsoft/TypeScript/issues/49462. Alternative build tools ocular-dev-tools or [microbundle](https://github.com/developit/microbundle) would avoid the issue, see https://publint.dev/ktx-parse@0.6.0 for example. But after discussion we're not inclined to swap out the build tool and will keep the `mv` for now, at least until the v9 migration settles down.